samples/openthread/cli/sample.yaml

```diff
@@ -31,7 +33,7 @@ tests:
       nrf21540dk_nrf52840
       nrf5340dk_nrf5340_cpuapp
       nrf5340dk_nrf5340_cpuapp_ns
-      nrf54l15pdk_nrf54l15_cpuapp
```

Comment:
Should that sample be extended to support both `nrf54l15pdk_nrf54l15_cpuapp` and `nrf54l15pdk_nrf54l15_cpuapp@0.3.0` as well?